tcms/static/js/testrun_actions.js

```diff
@@ -63,12 +63,24 @@ Nitrate.TestRuns.Details.on_load = function() {
     var case_id = c.find('input[name="case"]')[0].value;
     var case_run_id = c.find('input[name="case_run"]')[0].value;
     var case_text_version = c.find('input[name="case_text_version"]')[0].value;
-    var type = 'case_run';
     var callback = function(t) {
       // Observe the update case run stauts/comment form
+
+      const caseId = jQ('td > input[name=case_id]').val()
+      const executionId = jQ('td > input[name=object_pk]').val()
+
       c_container.parent().find('.update_form')
-        .unbind('submit').bind('submit', updateCaseRunStatus);
+        .unbind('submit').bind('submit', () => {
+          updateExecutionStatus(executionId, caseId, executionStatusPk)
```

Comment:
Note on names: You are using both `Id` and `Pk` in naming. Use one of the two but not both.